apps/portal/src/components/docs-page/example.tsx

```diff
@@ -20,9 +21,11 @@ import {
 
 type LiveProviderProps = ComponentProps<typeof LiveProvider>;
 
-export type ExampleProps = Pick<LiveProviderProps, "code" | "scope">;
+export type ExampleProps = Pick<LiveProviderProps, "code" | "scope"> & {
+  className?: string;
```

Comment:
Can we rename this to something like `contentClassName` or something to make it obvious it's applying to the content of the example block, not the block itself?